packages/tempus/src/Tempus_TimeStepControlStrategyPID.hpp

```diff
@@ -139,6 +138,7 @@ class TimeStepControlStrategyPID
      }
      tscsPL_->validateParametersAndSetDefaults(*this->getValidParameters());
 
+     strategy_name_ = tscsPL_->get<std::string>("Name");
```

Comment:
@sconde Why don't we need the "template" keyword here? I don't see a difference between this and what we had.